tests/integration/testdata/invoke/symlinked/src/app.py

```diff
@@ -0,0 +1,4 @@
+def handler(e, c):
+    with open("./linked-file.txt", "r") as file:
```

Comment:
This behaviour seems a little odd to me. This file is a directory up on the host machine, but in the container it's mounted in the same dir. I feel like this might make for confusing UX, or maybe I'm missing the use-case here/how it relates to build-in-source.

Reply:
The linked file here is actually one that lives up a folder level, testing if the link resolution behaviour is able to mount the file correctly
```
$ tree .
.
├── source-file.txt
├── src
│   ├── app.py
│   └── linked-file.txt -> ../source-file.txt
└── template.yaml
```
I can update this test to have a real world example of building + invoking a symlinked dependency folder since this that is more relevant to build in source.